docs/concepts.md

```diff
@@ -322,6 +322,66 @@ represented by a message containing two Channels handles:
     connection itself; this represents the actual authority of the caller.
   - an empty integrity component.
 
+### Signature labels
+
+Oak Wasm modules can be signed using [Ed25519](https://ed25519.cr.yp.to/)
+scheme. Such signatures may be created by an Oak module reviewer who then
+publishes signatures and corresponding public keys on a public resource. By
+signing a module reviewers endorse it's functionality and show that they trust
+this particular module not to have any undocumented functionality. The same
+reviewer may also use distinct keys for different purpose, for example to use a
+separate public key to sign a family of related modules.
+
+In case when a client trusts a certain reviewer, it can download a corresponding
```

Comment:
I don't think this will be a typical use case. The main use case would be where the end-user trusts the application developer, and the application developer ensures that the client application attaches the correct public key label to the data to ensure that it could be processed in future (e.g. after being stored) even if code must be updated (e.g. to fix a bug).

Reply:
Done
Added:
```
Reviewer's public key can also be assigned to outgoing requests by the client library, which was created by the
Oak application developer. In this case client doesn't need to manually download
a public key, but they will need to trust that the client library assigns these public keys to outgoing requests correctly.
```